docs/content/querying/lookups.md

```diff
@@ -292,3 +292,10 @@ To configure a Broker / Router / Historical / Peon to announce itself as part of
 |Property | Description | Default |
 |---------|-------------|---------|
 |`druid.lookup.tierName`| The tier for **lookups** for this node. This is independent of other tiers.|`__default`|
+
+
+## Introspect a Lookup
```

Comment:
Is the `POST` needed? from https://jersey.java.net/documentation/latest/jaxrs-resources.html#d0e2464 it seems that the sub-resource is request scoped, and would not require a POST to create it. Actually a POST shouldn't do anything since the return object should be invalid after the request has completed (If I'm reading jersey's jaxrs spec correctly)